ppml/trusted-big-data-ml/scala/docker-occlum/production/start-spark-local.sh

```diff
@@ -21,5 +21,5 @@ sudo docker run -it \
         -e CHALLENGE=cHBtbAo= \
         -e REPORT_DATA=ppml \
 	-e SGX_LOG_LEVEL=off \
-	intelanalytics/bigdl-ppml-trusted-big-data-ml-scala-occlum-production:2.4.0-SNAPSHOT-build \
+	intelanalytics/bigdl-ppml-trusted-big-data-ml-scala-occlum-production:2.4.0-build \
```

Comment:
could you update to `intelanalytics/bigdl-ppml-trusted-big-data-ml-scala-occlum-production:2.4.0`?